bucket/clash-for-windows.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://github.com/Fndroid/clash_for_windows_pkg",
+    "description": "A Windows/macOS GUI based on Clash",
+    "version": "0.13.1",
+    "license": "MIT",
+    "url": "https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.13.1/Clash.for.Windows.Setup.0.13.1.exe#/clash.7z",
+    "hash": "8ff7c72573fb8c2c620d815e6d7531825f7bd8fdfcae0fe135b7825150f6fdf9",
+    "shortcuts": [
+        [
+            "Clash for Windows.exe",
+            "Clash"
+        ]
+    ],
+    "pre_install": [
+        "Remove-Item -Recurse \"$dir\\Uninstall Clash for Windows.exe\"",
+        "Get-ChildItem -Path \"$dir\\`$PLUGINSDIR\" -Depth \"2\" -Include \"*.7z\" | Move-Item -Destination \"$dir\\`$PLUGINSDIR\"",
```

Comment:
Completely nonsense step when you can simply pass destination path to the expand-7zip

Reply:
> Completely nonsense step when you can simply pass destination path to the expand-7zip

Acturally, I'm not very familiar with this, and I just copy this from another manifest. I'll learn that.